x-pack/solutions/security/plugins/security_solution/public/asset_inventory/pages/all_assets.tsx

```diff
@@ -165,7 +168,7 @@ const AllAssets = ({
     onChangeItemsPerPage,
     setUrlQuery,
     onSort,
-    onResetFilters,
+    // onResetFilters,
```

Comment:
is this commented out intentionally? If so, consider adding a comment to explain why it’s left in the code and under what circumstances it might be re-enabled

Reply:
Yes, it will be re-enabled again soon, but giving a second thought, it might be better to just remove it for now